### PR TITLE
Fix Default Tech With Task Template

### PR DESCRIPTION
**Pull request by tsmr**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #0

When use task template without define a users_id_tech, the loaded template blank the users_id_tech field.

### CODE REVIEW

**Review by trasher — COMMENTED** [review]
I'm not sure this should be done in the ajax file. Also, a test case should be added.

File: ajax/task.php [hunk 1]
```diff
@@ -110,6 +110,9 @@
     }
 }
 
+if ($template->fields['users_id_tech'] == 0) {
+    $template->fields['users_id_tech'] = Session::getLoginUserID();
```
Comment (cedric-anne) [inline ajax/task.php:114]: If the previously selected tech is not the current user, I guess it will be replaced. This will still be a wrong behaviour.

**Review by cedric-anne — CHANGES_REQUESTED** [review]

File: ajax/task.php (hunk 1 above)
Comment (tsmr) [reply ajax/task.php]: You're right - an unset is better

**Review by cedric-anne — APPROVED** [review]

**Comment by cedric-anne** [comment]
> I'm not sure this should be done in the ajax file. Also, a test case should be added.

It should be placed either in this ajax file, either in this inline JS piece of code: https://github.com/glpi-project/glpi/blob/0a7f091f65f6ade62fbc376c1fe664bc4a24169d/templates/components/itilobject/timeline/form_task.html.twig#L510-L518

Both of them can only be tested with a end-to-end test and it is not possible on the 10.0/bugfixes branch yet.

**Review by trasher — APPROVED** [review]

**Pull request merged**